src/cli/clear.test.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import path from "path";
+
+import {makeClear} from "./clear";
+import {run} from "./testUtil";
+import {help} from "./clear";
+import * as Common from "./common";
+
+describe("cli/clear", () => {
+  const expectToPrint = (msg) =>
+    expect.arrayContaining([expect.stringMatching(msg)]);
+
+  describe("'help' command", () => {
+    it("prints usage when given no arguments", async () => {
+      expect(await run(help, [])).toEqual({
+        exitCode: 0,
+        stdout: expectToPrint(/^usage: sourcecred clear/),
+        stderr: [],
+      });
+    });
+
+    it("fails when given arguments", async () => {
+      expect(await run(help, ["foo/bar"])).toEqual({
+        exitCode: 1,
+        stdout: [],
+        stderr: expectToPrint(/^usage: sourcecred clear/),
+      });
+    });
+  });
+
+  describe("'clear' command", () => {
+    const mockFunc = () => jest.fn();
```

Comment:
mockFunc is equivalent to `jest.fn`. It doesn't serve any purpose that I can see.